website/docs/reference/resource-configs/bigquery-configs.md

```diff
@@ -484,8 +484,10 @@ when not matched then insert ...
 
 The `merge` approach has the benefit of automatically updating any late-arriving facts in the
 destination incremental table. The drawback of this approach is that BigQuery must scan all
-source tables referenced in the model SQL, as well as the entirety of the destination table.
-This can be slow and costly if the incremental model is transforming very large amounts of data.
+source tables referenced in the model SQL, as well as the the destination table (however the cost
```

Comment:
```suggestion
The 'merge' approach automatically updates new data in the destination incremental table but requires scanning all source tables referenced in the model SQL, as well as destination tables. This can be slow and expensive for large data volumes. [Partitioning and clustering](#using-table-partitioning-and-clustering) techniques mentioned earlier can help mitigate these issues.
```